dev/com.ibm.ws.org.jboss.weld3/bnd.overrides

```diff
@@ -3,18 +3,22 @@ bVersion=1.0
 
 Bundle-Name: weld OSGi bundle
 Bundle-SymbolicName: com.ibm.ws.org.jboss.weld3
-Bundle-Description: Weld OSGi Bundle, version 3.1.4.Final
+Bundle-Description: Weld OSGi Bundle, version 3.1.7.SP1
 Implementation-Title: Weld OSGi Bundle
 Bundle-License: http://www.apache.org/licenses/LICENSE-2.0.html
-Implementation-Version: 3.1.4.Final
+Implementation-Version: 3.1.7.SP1
 Embed-Dependency: *; scope=compile; inline=true
 Specification-Title: JSR-365 Contexts and Dependency Injection for Java
 Specification-Version: 2.0
 DynamicImport-Package: *
 Export-Package: !org.jboss.weld.environment.se.*, \
- org.jboss.*;version="3.1.4.Final", \
- org.jboss.weldx.transaction;version="1.0.16"
-Private-Package: org.jboss.weld.environment.se.*
+ org.jboss.*;version="3.1.7.SP1", \
+ org.jboss.weldx.transaction;version="1.0.16", \
+ !org.jboss.weld.bean.proxy.util\
+ !org.jboss.weld.proxy
```

Comment:
Why this change. Were these packages unexported in the 3.1.7.SP1 bundle manifest?

Reply:
I think that was just an attempt to cut the number of exports.